bundles/at.medevit.elexis.text.docx/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,40 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Docx
+Bundle-SymbolicName: at.medevit.elexis.text.docx;singleton:=true
+Bundle-Version: 3.10.0.qualifier
+Bundle-Vendor: medevit.at
+Automatic-Module-Name: at.medevit.medelexis.text.docx
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: ch.rgw.utility;bundle-version="3.9.100",
+ ch.elexis.core.data;bundle-version="3.9.0",
+ ch.elexis.core.ui;bundle-version="3.9.0",
+ org.apache.commons.lang3;bundle-version="3.12.0",
+ org.apache.commons.io;bundle-version="2.8.0",
+ org.apache.commons.codec;bundle-version="1.14.0",
+ com.sun.xml.bind.jaxb-osgi;bundle-version="2.3.4"
+Import-Package: javax.xml.bind,
+ javax.xml.bind.annotation,
+ javax.xml.bind.util,
+ org.apache.commons.logging,
+ org.slf4j
+Bundle-ClassPath: .,
+ lib/antlr-runtime-3.5.2.jar,
```

Comment:
die jars in `lib`

1) das versteh nicht, sollten die nicht teil des commits sein?

2) könnte man hier ggf. welche über das target reinnehmen die jetzt manuell dabei sin?

Reply:
Okay. Da im .gitignore alle *.jar-Dateien ignoriert wurden, sind die bei mir durchgefallen.
Habe die jars und das .gitignore in separatem commit hinzugefügt.